### PR TITLE
Add a hover on Show Person Avatar 

### PR DESCRIPTION
**Pull request by PepeGonzale**
#1149 

- Added onClick prop to Avatar.tsx
- Added a onClick console.log to test in PersonShow.tsx
- Added some styles on hover

### CODE REVIEW

File: front/src/modules/ui/layout/show-page/components/ShowPageSummaryCard.tsx [hunk 1]
```diff
@@ -98,6 +98,7 @@ export const ShowPageSummaryCard = ({
       <StyledAvatarWrapper onClick={onAvatarClick}>
         <Avatar
           avatarUrl={logoOrAvatar}
+          onClick={() => console.log('on click')}
```
Comment (charlesBochet) [inline front/src/modules/ui/layout/show-page/components/ShowPageSummaryCard.tsx:101]: can we have this method trigger the existing onAvatarClick method and remove the onClick on StyledAvatarWrapper (also onAvatarClick should be renamed handleAvatarClick)

File: front/src/modules/ui/view-bar/components/UpdateViewButtonGroup.tsx [hunk 2]
```diff
@@ -28,13 +28,20 @@ const StyledContainer = styled.div`
   margin-right: ${({ theme }) => theme.spacing(2)};
   position: relative;
 `;
-
 export type UpdateViewButtonGroupProps = {
+<<<<<<< HEAD
```
Comment (charlesBochet) [inline front/src/modules/ui/view-bar/components/UpdateViewButtonGroup.tsx:32]: conflicts!

File: front/src/modules/users/components/Avatar.tsx [hunk 3]
```diff
@@ -64,6 +68,7 @@ const StyledAvatar = styled.div<AvatarProps & { colorId: string }>`
     }
   }};
   justify-content: center;
+  transition: border 0.2s ease;
```
Comment (charlesBochet) [inline front/src/modules/users/components/Avatar.tsx:71]: let's use the theme for transition duration :) (see animation.ts)

**Review by charlesBochet — COMMENTED** [review]
Thank you, left a few comments!

**Comment by PepeGonzale** [comment]
> Thank you, left a few comments!

I will change it!

File: front/src/modules/ui/view-bar/components/ViewBar.tsx [hunk 4]
```diff
@@ -55,7 +55,13 @@ export const ViewBar = ({
           rightComponent={
             <UpdateViewButtonGroup
               onViewEditModeChange={openOptionsDropdownButton}
+<<<<<<< HEAD
```
Comment (charlesBochet) [inline front/src/modules/ui/view-bar/components/ViewBar.tsx:58]: still have conflict here!

File: front/src/modules/users/components/Avatar.tsx [hunk 5]
```diff
@@ -64,6 +68,7 @@ const StyledAvatar = styled.div<AvatarProps & { colorId: string }>`
     }
   }};
   justify-content: center;
+  transition: border ${props => props.theme.animation.duration.normal} ease;
```
Comment (charlesBochet) [inline front/src/modules/users/components/Avatar.tsx:71]: You can run "yarn lint" to catch these errors!

File: front/src/modules/users/components/Avatar.tsx [hunk 6]
```diff
@@ -79,13 +84,19 @@ const StyledAvatar = styled.div<AvatarProps & { colorId: string }>`
         return '12px';
     }
   }};
+
+  &:hover {
+    border-color: ${({ onClick }) =>
+      onClick ? 'rgba(0, 0, 0, 0.1)' : 'transparent'};
```
Comment (thaisguigon) [inline front/src/modules/users/components/Avatar.tsx:90]: The issue mentions a "transparent lighter" color, so here I assume we would like to use `theme.background.transparent.lighter`. Is that correct @Bonapara ?

**Review by thaisguigon — CHANGES_REQUESTED** [review]

File: front/src/modules/users/components/Avatar.tsx (hunk 6 above)
Comment (Bonapara) [reply front/src/modules/users/components/Avatar.tsx:90]: Correct for the color! The border weight should be 4px

Comment (Bonapara) [reply front/src/modules/users/components/Avatar.tsx:90]: Thanks @thaisguigon!

File: front/src/modules/users/components/Avatar.tsx [hunk 7]
```diff
@@ -26,8 +27,11 @@ const StyledAvatar = styled.div<AvatarProps & { colorId: string }>`
     isNonEmptyString(avatarUrl) ? `background-image: url(${avatarUrl});` : ''}
   background-position: center;
   background-size: cover;
+  border: ${({ onClick }) => (onClick ? '1px' : '0')};
```
Comment (charlesBochet) [inline front/src/modules/users/components/Avatar.tsx:30]:
```suggestion
```

File: front/src/modules/users/components/Avatar.tsx [hunk 8]
```diff
@@ -26,8 +27,11 @@ const StyledAvatar = styled.div<AvatarProps & { colorId: string }>`
...
+  border-color: transparent;
```
Comment (charlesBochet) [inline front/src/modules/users/components/Avatar.tsx:31]:
```suggestion
```

File: front/src/modules/users/components/Avatar.tsx [hunk 9]
```diff
@@ -79,13 +84,19 @@ const StyledAvatar = styled.div<AvatarProps & { colorId: string }>`
         return '12px';
     }
   }};
+
+  &:hover {
+    border-color: ${({ onClick }) =>
```
Comment (charlesBochet) [inline front/src/modules/users/components/Avatar.tsx:89]:
```suggestion
    box-shadow: 0 0 0 4px ${theme.background.transparent.light};
```

File: front/src/modules/users/components/Avatar.tsx (hunk 6 above)
Comment (charlesBochet) [inline front/src/modules/users/components/Avatar.tsx:90]:
```suggestion
```

**Review by charlesBochet — APPROVED** [review]
Thank you! Applying @thaisguigon feedbacks!

File: front/src/modules/users/components/Avatar.tsx [hunk 10]
```diff
@@ -64,6 +66,7 @@ const StyledAvatar = styled.div<AvatarProps & { colorId: string }>`
     }
   }};
   justify-content: center;
+  transition: border ${(props) => props.theme.animation.duration.normal} ease;
```
Comment (charlesBochet) [inline front/src/modules/users/components/Avatar.tsx:69]:
```suggestion
```

**Review by charlesBochet — APPROVED** [review]

**Pull request merged**